fedimint-core/src/admin_client.rs

```diff
@@ -94,6 +85,64 @@ impl WsAdminClient {
             .await
     }
 
+    /// Gets the default config gen params which can be configured by the
+    /// leader, gives them a template to modify
+    pub async fn get_default_config_gen_params(&self) -> FederationResult<ConfigGenParamsRequest> {
+        self.request_auth(
+            "/get_default_config_gen_params",
```

Comment:
Not something we need to address here, but we really need to get rid of that `/`. These are all jsonrpc calls, and not urls, so why are we doing slashes? :D

Also if we are to keep with the convnentions that bitcoind or LN nodes use, we should probably name `getdefaultconfiggenparams`? I know it looks less readable, but bitcoin uses `getblockcount` and not `get_block_count`.

Ping @Maan2003 for awareness and to tell me in case I'm wrong.

Reply:
Agreed, lets not add new `/`s.